### PR TITLE
Fix lib.qt soft

### PR DESCRIPTION
**Pull request by dsavenko**
Hi,

This PR fixes `lib.qt` soft. After the PR, CK will be able to correctly find Qt and provide necessary variables for compilation and running of Qt programs.

Tested on Ubuntu, Raspberry Pi and Windows 10 with Qt5 (various subversions) and some of Qt4.

### CODE REVIEW

File: soft/lib.qt/.cm/meta.json [hunk 1]
```diff
@@ -5,16 +5,20 @@
...
     "soft_file": {
-      "linux": "libqtcore.so",
-      "win": "Qt*Core.lib"
+      "linux": "qmake",
+      "win": "qmake.exe"
     },
     "soft_path_example": {
-      "linux": "/usr/lib/x86_64-linux-gnu/Qt/.so",
-      "linux": "C:\\Qt\\5.8\\msvc2015_64\\lib\\Qt5Core.lib"
+      "linux": "/usr/lib/x86_64-linux-gnu/qt5/bin/qmake",
```
Comment (DVEfremov) [inline soft/lib.qt/.cm/meta.json]: at my ubunty with qmake 
```daniil@daniil:~$ sudo ls -l /usr/lib/x86_64-linux-gnu/qt5/bin/qmake
ls: cannot access '/usr/lib/x86_64-linux-gnu/qt5/bin/qmake': No such file or directory
daniil@daniil:~$ qmake --version
QMake version 2.01a
Using Qt version 4.8.7 in /usr/lib/x86_64-linux-gnu
```

Comment (dsavenko) [inline soft/lib.qt/.cm/meta.json]: What command does output this? This line in the meta is just an example, it isn't really used anywhere for Qt recognition.

Comment (dsavenko) [inline soft/lib.qt/.cm/meta.json]: Please try `ck detect soft:lib.qt`. Does it work?

Comment (DVEfremov) [inline soft/lib.qt/.cm/meta.json]: it was just 
ls -l /usr/lib/x86_64-linux-gnu/qt5/bin/qmake
but may be this my local specific problem.

**Pull request merged**